src/govuk/components/task-list/task-list.yaml

```diff
@@ -0,0 +1,212 @@
+params:
```

Comment:
@frankieroberto I think you're missing the `hint` params from the docs.

Reply:
For example:

```
  - name: hint.text
    type: string
    required: true
    description: If `html` is set, this is not required. Text to use within the hint. If `html` is provided, the `text` argument will be ignored.
  - name: hint.html
    type: string
    required: true
    description: If `text` is set, this is not required.
```

Reply:
Opps, well spotted! 🙌